### PR TITLE
Add documentation for the fujitsu_hvac integration

### PR DESCRIPTION
**Pull request by crevetor**
## Proposed change
Add documentation for the new fujitsu_hvac integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109335
- Link to parent pull request in the Brands repository:  https://github.com/home-assistant/brands/pull/5159
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for integrating Fujitsu heat pumps with the Ayla IoT platform, enhancing climate control capabilities.
	- Added detailed guidelines for managing Fujitsu heat pumps and air conditioners through the FGLair application, streamlining user setup and interaction with their devices.

### CODE REVIEW

**Comment by crevetor** [comment]
I wasn't sure what to put in the ha_release field since this hasn't been merged yet. Let me know and I will change it.

File: source/_integrations/fujitsu_fglair.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ha_release: 2024.9.2
+ha_domain: fujitsu_fglair
+ha_integration_type: integration
+ha_codeowners:
+  - '@crevetor'
+ha_config_flow: true
+ha_platforms:
+  - climate
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: gold
```
Comment (joostlek) [inline source/_integrations/fujitsu_fglair.markdown]:
```suggestion
```

File: source/_integrations/fujitsu_fglair.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Fujitsu FGLair
+description: Control your Fujitsu heat pump or air conditioner that uses the FGLair app
+ha_category:
+  - Climate
+ha_release: 2024.9.2
```
Comment (joostlek) [inline source/_integrations/fujitsu_fglair.markdown]:
```suggestion
ha_release: 2024.9
```

File: source/_integrations/fujitsu_fglair.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ha_codeowners:
+  - '@crevetor'
+ha_config_flow: true
+ha_platforms:
+  - climate
+ha_iot_class: "Cloud Polling"
+---
+
+The {{ page.title }} {% term integration %} provides support for Fujitsu heat pumps and air conditioners that use the FGLair app.
+You can check [this link](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html) to know which app to use for your heat pump.
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown]:
```suggestion
To find out which app to use for your heat pump, check [the Fujitsu FGLair FAQ](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html).
```

File: source/_integrations/fujitsu_fglair.markdown [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Prerequisite
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/fujitsu_fglair.markdown [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+The {{ page.title }} {% term integration %} provides support for Fujitsu heat pumps and air conditioners that use the FGLair app.
+To find out which app to use for your heat pump, check [the Fujitsu FGLair FAQ](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html).
+
+## Prerequisites
+
+First, set up your device in the FGLair app before using this integration.
+To configure this integration, you will need the credentials (login and password) used to connect to the FGLair application.
+
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown:25]: Is there a list of supported devices or models?

Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown:25]: Could you add a section ## Climate, listing which functions are supported? Like Gree does, for example.

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/fujitsu_fglair.markdown (hunk 5 above)
Comment (crevetor) [reply source/_integrations/fujitsu_fglair.markdown:25]: I haven't been able to find one.  I have a list of supported WLAN adapter models. As far as I understand it you can use 2 different types of WLAN adapters on the same heat pump model which would mean that the model is supported by this integration if you use one WLAN adapter model and by another if you choose the other one.

File: source/_integrations/fujitsu_fglair.markdown [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+This integration supports the following functionalities (if the devices support them):
+
+- [`set_hvac_mode`](/integrations/climate/#action-climateset_hvac_mode)
+- [`target temperature`](/integrations/climate#action-climateset_temperature)
+- [`turn on/off`](/integrations/climate#action-climateturn_on)
+- [`fan mode`](/integrations/climate#action-climateset_fan_mode)
+- [`swing mode`](/integrations/climate#action-climateset_swing_mode)
+
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown]:
```suggestion
```

File: source/_integrations/fujitsu_fglair.markdown [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+The {{ page.title }} {% term integration %} provides support for Fujitsu heat pumps and air conditioners that use the FGLair app.
+To find out which app to use for your heat pump, check [the Fujitsu FGLair FAQ](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html).
+
+## Prerequisites
+
+First, set up your device in the FGLair app before using this integration.
+To configure this integration, you will need the credentials (login and password) used to connect to the FGLair application.
+
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown:24]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/fujitsu_fglair.markdown [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+ha_iot_class: "Cloud Polling"
+---
+
+The {{ page.title }} {% term integration %} provides support for Fujitsu heat pumps and air conditioners that use the FGLair app.
+To find out which app to use for your heat pump, check [the Fujitsu FGLair FAQ](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html).
+
+## Prerequisites
+
+First, set up your device in the FGLair app before using this integration.
+To configure this integration, you will need the credentials (login and password) used to connect to the FGLair application.
```
Comment (c0ffeeca7) [inline source/_integrations/fujitsu_fglair.markdown:23]:
```suggestion
To configure this integration, you will need the credentials (login and password) used to connect to the FGLair application.

```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @crevetor :+1:

**Pull request merged**